src/material/slider/_slider-theme.scss

```diff
@@ -49,6 +49,11 @@
     theming.get-color-from-palette($foreground, base, $mat-slider-tick-opacity);
   $mat-slider-tick-size: 2px;
 
+  .mat-slider-wrapper {
+    -webkit-print-color-adjust: exact;
+    color-adjust: exact;
```

Comment:
Can you leave a comment saying why this is necessary?

Reply:
It forces the browser to include the background color when the print function is used. I didn't include a media query for printing because the spec is not specifically intended for printing and a slider without the background-color doesn't make any sense. 

Before:
![image](https://user-images.githubusercontent.com/62592273/111909993-c82f4780-8a5f-11eb-994f-fbfe1a7769d5.png)

After:
![image](https://user-images.githubusercontent.com/62592273/111910014-e39a5280-8a5f-11eb-815f-5fc10c2def6f.png)

Reply:
Yes, I'm aware what it does, but we should have a comment in the code so it doesn't get deleted. Also I think that this should be in `_slider.scss`, not `_slider-theme.scss`.

Reply:
Ah! I updated the pull request.